src/test/wix/TestData/WixprojPackageVcxprojWindowsApp/WixprojPackageVcxprojWindowsApp.wixproj

```diff
@@ -5,5 +5,7 @@
     <ProjectReference Include="..\VcxprojWindowsApp\VcxprojWindowsApp.vcxproj" />
     <ProjectReference Include="..\WixprojLibraryVcxprojDll\WixprojLibraryVcxprojDll.wixproj" />
     <ProjectReference Include="..\WixprojModuleCsprojWinFormsNetFx\WixprojModuleCsprojWinFormsNetFx.wixproj" />
+
+    <PackageReference Include="Wixtoolset.UI.wixext" Version="4.0.0-preview.1-build.205" />
```

Comment:
I hope the build fails because of this hardcoded version.

Reply:
Interesting. The test did not fail. There is probably a warning in the test's build indicating that a higher version was selected. I'm going to try to make things a little better to handle the tests under the `build` folder.

Reply:
It succeeded on AppVeyor, too. So you could merge this now and fix it later if you want.

Reply:
Lets' see if this last commit makes everything work consistently.